<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_SoftwareFirstTip</class>
 <widget class="QDialog" name="W_SoftwareFirstTip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>提示</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget_title{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_context" native="true">
     <property name="styleSheet">
      <string notr="true">#widget_context{
	background-color: rgb(255, 255, 255);
	border-bottom:1px solid #aaa;
}
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,0" columnstretch="2,1">
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>
如果你是第一次使用该工具，那么先去看看帮助文档。
了解该工具的主要功能哦！</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="toolButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>帮助文档</string>
        </property>
        <property name="icon">
         <iconset resource="../../../drillgifactionsequenceeditor.qrc">
          <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/nav/Help.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/nav/Help.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>不再提醒</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../drillgifactionsequenceeditor.qrc"/>
 </resources>
 <connections/>
</ui>
